lib/activerecord-import/adapters/postgresql_adapter.rb

```diff
@@ -21,7 +21,7 @@ def insert_many( sql, values, *args ) # :nodoc:
       insert( sql2insert, *args )
     end
 
-    ActiveRecord::Base.connection.query_cache.clear
+    query_cache.clear
```

Comment:
Yes, I think that would be much better! Could you also take a look at the test and make sure it's actually testing what we want. It doesn't look to me like the existing test is useful.